### ISSUE TITLE
feature request: Simplisafe

### ISSUE THREAD

**Issue by thesle3p** [furrtek/portapack-havoc#304]
Would it be possible to add support for Simplisafe? Simplisafe is a common DiY Alarm system that uses unencrypted 433.92 MHz with ASK  Modulation with PiWM for communication,  A module for it already exists in rtl_433 (https://github.com/merbanan/rtl_433/blob/master/src/devices/simplisafe.c)

**Comment by eried** [comment]
Yes, anything is possible, but nobody is really going to add something they do not use. I would suggest you to check the code and hack something copying one of the actual apps. I personally haven't learned it yet (im not a radio guy, or low level coder) but your request will be only fullfill if you do it.
